src/views/states/list/components/StateRow.tsx

```diff
@@ -19,16 +21,18 @@ const StateRow: FC<RowProps<V1beta1NodeNetworkState, { rowIndex: number }>> = ({
 }) => {
   const [isExpanded, setIsExpanded] = useState(false);
   const { t } = useNMStateTranslation();
-  const interfaces = obj?.status?.currentState?.interfaces;
+  const interfaces = obj?.status?.currentState?.interfaces as NodeNetworkConfigurationInterface[];
+
+  const filteredInterfaces = useFilteredInterfaces(interfaces);
 
-  const interfacesByType = interfaces?.reduce((acc, iface) => {
+  const interfacesByType = filteredInterfaces?.reduce((acc, iface) => {
```

Comment:
this will also re render a lot?

Reply:
Should i `useMemo` in `StateRow`?

Reply:
`useMemo` in the `useFilteredInterfaces` and `reduce`?

Reply:
Maybe better `useFilteredInterfaces` and `reduce` in `useMemo` ? But I can see you are using `useMemo` in `useInterfaceFilters` used in `useFilteredInterfaces`, so maybe it's enough?